bot/exts/moderation/infraction/infractions.py

```diff
@@ -351,9 +355,13 @@ async def apply_voice_ban(self, ctx: Context, user: UserSnowflake, reason: t.Opt
         if reason:
             reason = textwrap.shorten(reason, width=512, placeholder="...")
 
-        await user.move_to(None, reason="Disconnected from voice to apply voiceban.")
+        action = None
+
+        # Skip members that left the server
+        if isinstance(user, Member):
+            await user.move_to(None, reason="Disconnected from voice to apply voiceban.")
+            action = user.remove_roles(self._voice_verified_role, reason=reason)
```

Comment:
This was actually not implemented correctly originally. Both of these functions should be the action, not just `remove_roles`. The benefit of being an action is that `apply_infraction` will do error handling if the action fails.